annual_meantemp.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
+    "title": "Regional average climate observations UK Annual Mean Temperature",
+    "summary": "Climate observations that have been averaged across UK geographic areas",
+    "description": "Derived from https://catalogue.ceda.ac.uk/uuid/4dc8450d889a491ebb20e724debe2dfb",
+    "dataset_version": "v1.1.0.0",
+    "publisher": "https://www.gov.uk/government/organisations/met-office",
+    "creator": "https://www.gov.uk/government/organisations/met-office",
+    "themes": "http://gss-data.org.uk/def/gdp#climate-change",
+    "license": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+    "keywords": [
+        "climate",
+        "temperature"
+    ],
+    "columns": {
+        "Year": {
+            "from_template": "year"
+        },
+        "Geography": {
+            "label": "Met Office Geography",
+            "from_existing": "http://purl.org/linked-data/sdmx/2009/dimension#refArea",
+            "type": "dimension"
+        },
+        "Measure": {
+            "type": "measures",
+            "values": [
+                {
+                    "label": "Mean",
+                    "description": "The annual mean temperature, averaged out for the reporting period"
+                },
+                {
+                    "label": "Trend",
+                    "description": "Climatological trend calculated using triangular window method for each geographical area"
+                }
+            ]
+        },
+        "Value": {
+            "type": "observations",
+            "unit": {
+                "label": "Celsius",
```

Comment:
Remove the label, it's implied when you use Celsius directly.

Reply:
Label removed for value unit.